elements/lisk-cryptography/test/protocol_specs/bls_specs/fast_aggregate_verify/pk_is_identity_element.yml

```diff
@@ -0,0 +1,7 @@
+input:
```

Comment:
I find the file name slightly misleading as one could think that one public key is the identity element, but it is about the aggregate public key. So either
"sum_of_pks_is_identity_element" or
"aggregate_pk_is_identity_element"